### PR TITLE
Documented arguments' API

### PR DESCRIPTION
**Pull request by mariomac**
#### Description of the changes

Documented arguments' API.

### Author

- [ ] add a risk label after carefully considering the "blast radius" of your changes
- [ ] describe the _intent_ of your changes in the description. don't just rewrite your code in prose
- [ ] assign at least one reviewer

### Reviewer

- [ ] review code for readability
- [ ] verify that high risk behavior changes are well tested
- [ ] check license for any new external dependency
- [ ] ask questions about anything that isn't clear and obvious
- [ ] approve the PR when you consider it's good to merge

### CODE REVIEW

File: docs/toolset/args.md [hunk 1]
```diff
@@ -1,3 +1,106 @@
...
+```
+
+Invoking `argsTest` without arguments should show the default values:
+
+```
+{DefaultArgumentList:{Verbose:false Pretty:false All:false Metrics:false
+Inventory:false Events:false} SomeInt:1000 SomeString:hello}
+```
+
+You can combine command-line with environment arguments, giving priority to command-line:
```
Comment (varas) [inline docs/toolset/args.md]: Actually the environment has priority over the CLI

Comment (varas) [reply docs/toolset/args.md]: https://github.com/newrelic/infra-integrations-sdk/blob/master/args/args_test.go#L115

Comment (mariomac) [reply docs/toolset/args.md]: Yes, I know... I was just advancing the documentation to what we will do soon.

File: docs/toolset/args.md [hunk 2]
```diff
@@ -1,3 +1,106 @@
...
+## Bundled arguments
+
+The arguments API provides a `DefaultArgumentList` struct that contains all the arguments any integration should
+consider. All of them are `bool` and default to `false`. They are described following, named by its
+[Go code convention](#naming-format-convention):
+
+* `Events`: whether the integration should publish events' data.
+* `Inventory`: whether the integration should publish inventory data.
+* `Metrics`: whether the integration should publish metrics' data.
+* `All`: whether the integration should publish events, inventory and metrics data.
```
Comment (varas) [inline docs/toolset/args.md]: `All` has been removed

**Pull request merged**